static/variables/ui-variables.less

```diff
@@ -82,4 +82,4 @@
 
 // Other
 
-@font-family: '.SFNSText-Regular', 'SF UI Text', 'Lucida Grande', 'Segoe UI', Ubuntu, Cantarell, sans-serif;
+@font-family: 'system';
```

Comment:
Is it safe to use system as name?

I just read some adventure about the system font: https://medium.com/@mwichary/system-shock-6b1dc6d6596f#.v0lk6vj4x